.github/workflows/security-scan.yml

```
@@ -0,0 +1,87 @@
+name: dynamic-security-scan
+
+on:
+  schedule:
+    - cron: "45 8 * * 1" 
+  workflow_dispatch:
+
+env:
+  APPLICATION: submit
+  DOCKER_REPO: ${DOCKER_REPO}/submit:latest
+  ZAP_VERSION: 2.15.0
+
+jobs:
+  dynamic-audit:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_APPLICATION_TAG: latest
+    outputs:
+      alerts_info: ${{ steps.report.outputs.alerts_info }}
+      alerts_low: ${{ steps.report.outputs.alerts_low }}
+      alerts_medium: ${{ steps.report.outputs.alerts_medium }}
+      alerts_high: ${{ steps.report.outputs.alerts_high }}
+      alerts_total: ${{ steps.report.outputs.alerts_total }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup dependencies
+        run: |
+          npm install
```

```suggestion
          npm ci
```

`npm ci` might be better.

Ref: https://stackoverflow.com/a/53325242